pymc_experimental/model_transform/conditioning.py

```diff
@@ -206,3 +213,132 @@ def do(
     if prune_vars:
         return prune_vars_detached_from_observed(model)
     return model
+
+
+def change_value_transforms(
+    model: Model,
+    vars_to_transforms: Dict[ModelVariable, Union[RVTransform, None]],
+) -> Model:
+    """Change the value variables transforms in the model
+
+    Parameters
+    ----------
+    model: Model
+    vars_to_transforms: Dict
+        Mapping between RVs and new transforms to be applied to the respective value variables
+
+    Returns
+    -------
+    new_model: Model
+        Model with the updated transformed value variables
+
+    Examples
+    --------
+    Extract untransformed space Hessian after finding transformed space MAP
+
+    .. code-block:: python
+
+        import pymc as pm
+        from pymc.distributions.transforms import logodds
+        from pymc_experimental.model_transform.conditioning import change_value_transforms
+
+        with pm.Model() as base_m:
+            p = pm.Uniform("p", 0, 1, transform=None)
+            w = pm.Binomial("w", n=9, p=p, observed=6)
+
+        with change_value_transforms(base_m, {"p": logodds}) as transformed_p:
+            mean_q = pm.find_MAP()
+
+        with change_value_transforms(transformed_p, {"p": None}) as untransformed_p:
+            new_p = untransformed_p['p']
+            std_q = ((1 / pm.find_hessian(mean_q, vars=[new_p])) ** 0.5)[0]
+
+        print(f"  Mean, Standard deviation\np {mean_q['p']:.2}, {std_q[0]:.2}")
+        #   Mean, Standard deviation
+        # p 0.67, 0.16
+
+    """
+    vars_to_transforms = {
+        parse_vars(model, var)[0]: transform for var, transform in vars_to_transforms.items()
+    }
+
+    if set(vars_to_transforms.keys()) - set(model.free_RVs):
+        raise ValueError(f"All keys must be free variables in the model: {vars_to_transforms}")
```

Comment:
```suggestion
        raise ValueError(f"All keys must be free variables in the model: {set(vars_to_transforms)}")
```

Reply:
printing `model.free_RVs`. I don't want to make it a set so that the error message doesn't confusingly change when the same snippet is rerun